src/v2/view-builder/views/IdentifierView.js

```diff
@@ -160,6 +164,25 @@ const Body = BaseForm.extend({
     return newSchemas;
   },
 
+  showCustomFormErrorCallout(error, messages) {
+    if (!error?.responseJSON?.errorSummaryKeys?.includes(CUSTOM_ACCESS_DENIED_KEY)) {
+      return false;
+    }
+
+    const { errorSummary } = error.responseJSON;
+
+    const options = {
+      type: 'error',
+      content: new CustomAccessDeniedErrorMessage({
+        message: errorSummary,
+        links: messages[0]?.links,
```

Comment:
There are cases where the [IDX response will contain more than one message](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/errors-authenticator-verification-okta-verify-reject-push.json#L8-L38), will that ever happen with a custom access denied error message?

Reply:
Approved PR so you're not stuck, but would consider whether the multi-message response scenario could apply. If the custom error message is not the first message in the response, it'll also cause an incorrect render (raw html, again)

Reply:
I'll grab the message with the right key